src/emulator/renderer/src/texture_cache.cpp

```diff
@@ -94,10 +94,14 @@ void cache_and_bind_texture(TextureCacheState &cache, const SceGxmTexture &gxm_t
     const TextureCacheHash hash = hash_texture_data(gxm_texture, mem);
 
     // Try to find GXM texture in cache.
-    const TextureCacheGxmTextures::const_iterator gxm_begin = cache.gxm_textures.cbegin();
-    const TextureCacheGxmTextures::const_iterator gxm_end = gxm_begin + cache.used;
-    const TextureCacheGxmTextures::const_iterator cached_gxm_texture = std::find(gxm_begin, gxm_end, gxm_texture);
-    if (cached_gxm_texture == gxm_end) {
+    size_t cached_gxm_texture_index = -1;
+    for (size_t a = 0; a < cache.used; a++) {
```

Comment:
Reason for changing from std::find

Reply:
Explained in #developement but for some reason the == operator got deleted. I couldn't compare both structs directly, so it failed to compile.